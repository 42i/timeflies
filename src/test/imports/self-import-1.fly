import self-import-1.fly

day 2012-01-03 8 12
; aaa 2.5
; bbb 1.5