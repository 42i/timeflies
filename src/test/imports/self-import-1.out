imports/self-import-1.fly:1: WARNING : file imports/self-import-1.fly already processed
Time at work overview (all):
     when        worked   leave    sick balance
2012-01-03 Tue:    4.00 ----.-- ----.--   -4.00
              ; aaa 2.5
              ; bbb 1.5
  week 2012-01:    4.00 ----.-- ----.--   -4.00
 month 2012-01:    4.00 ----.-- ----.--   -4.00
         total:    4.00 ----.-- ----.--   -4.00
     when        worked   leave    sick balance
